specification/appconfiguration/AppConfiguration/main.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "./routes.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@service({
+  title: "Azure App Configuration",
+})
+@versioned(Versions)
+// FIXME: Check API key header name and find service scope
+@useAuth(
+  ApiKeyAuth<ApiKeyLocation.header, "Connection String"> | OAuth2Auth<[
```

Comment:
@avanigupta please confirm if this is the correct header for your api key auth?

Reply:
Connection strings are provided via `Authorization` header. Additionally, the following headers are also needed for authentication: https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-authentication-hmac#prerequisites

Reply:
I went back to look at existing SDKs and actually do see connection strings supported through the "Connection String" header: https://github.com/Azure/azure-sdk-for-python/blob/6c41eea8b77f907115a854a7c490d61cc40071f1/sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_generated/_configuration.py#L65 going to leave as is since it also matches your previous specification